spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/util/RetryUtil.scala

```diff
@@ -0,0 +1,21 @@
+package com.datastax.spark.connector.util
+
+import scala.annotation.tailrec
+import scala.util.{Failure, Success, Try}
+
+object RetryUtil {
```

Comment:
Would [Eventually](http://www.artima.com/docs-scalatest-2.0.M5/org/scalatest/concurrent/Eventually.html) be ok instead?